test/js/strings-test.js

```diff
@@ -23,6 +23,13 @@ test('FromV8String', function (t) {
   t.equal(a('an utf8 strïng'), 'an utf8 strïng');
   t.equal(b('an utf8 strïng'), 'an utf8 strïng');
 
-  t.equal(bindings.encodeHex(), new Buffer('hello').toString('hex'));
+  var buf;
+
+  if (typeof(Buffer.from) === "function") {
```

Comment:
In old Node versions (at least some 4.x) this will be true because the base class `Uint8Array` exposes `from` which is not the same as `Buffer.from` in actual Node versions.
But you can check on `Buffer.alloc` instead.

Reply:
Thanks for that.   I was just scratching my head because there was an unexpected Travis CI failure against node 0.10 on macos, only.